specifications/13-constraint-operators.json

```diff
@@ -325,6 +325,47 @@
                         ]
                     }
                 ]
+            },
+            {
+                "name": "F7.date-with-timezone",
+                "description": "dates",
+                "enabled": true,
+                "strategies": [
+                    {
+                        "name": "default",
+                        "parameters": {},
+                        "constraints": [
+                            {
+                                "contextName": "currentTime",
+                                "operator": "DATE_AFTER",
+                                "value": "2022-01-22T13:00:00.000+02:00"
+                            },
+                            {
+                                "contextName": "currentTime",
+                                "operator": "DATE_BEFORE",
+                                "value": "2022-01-22T14:00:00.000+02:00"
```

Comment:
Doesn't RFC3339 also support negative time zone offsets? Should we test for that explicitly? It's less common, so might catch more people off guard 💁🏼

Reply:
Yeah, it does, I think this is just a product of our timezones, since we're GMT+1/GMT+2, I suspect folks the west of GMT are using these regularly. This test is mostly just to ensure that that some attempt to respect timezones is in place, I don't think we should create exhaustive tests here, that should be done closer to the SDK implementation